plot_krun_results.py

```diff
@@ -378,31 +388,196 @@ def read_krun_results_file(results_file):
     return None
 
 
-def get_data_dictionaries(json_files):
+def get_data_dictionaries(json_files, benchmarks=[], outliers=False,
+                          unique_outliers=False):
     """Read a list of BZipped JSON files and return their contents as a
     dictionaries of machine name -> JSON values.
+
+    This function returns ONLY the data that the user has requested on the
+    command line. Therefore, we check carefully that all data can be found
+    in the available Krun results files. Also, we pass back the title text
+    for each subplot.
     """
-    data_dictionary = dict()
+
+    data_dictionary = { 'data': dict(), # Measurement data to be plotted.
+                        'all_outliers': dict(),
+                        'common_outliers': dict(),
+                        'unique_outliers': dict(),
+                       }
```

Comment:
Yeah, it's a space to the right of the opening brace. Do you normally line the braces up or align the closing brace to the first `d` of `data_dictionary`?